files/zh-cn/learn/javascript/building_blocks/functions/index.md

```diff
@@ -27,17 +27,19 @@ slug: Learn/JavaScript/Building_blocks/Functions
 
```

Comment:
Please update line#15-#18:

```suggestion
      <td>
        基本的电脑知识，对 HTML 与 CSS 有基本的了解，及已阅读：<a href="/zh-CN/docs/Learn/JavaScript/First_steps"
          >JavaScript 第一步</a>。
      </td>
```
We usually only add translator's notes for sentences or words that are easily ambiguous instead of directly translating them.

If it is necessary for some glossary, we usually:
- add a note for the first appearance in the context of the word, if its concept was clearly in web development but has no authoritative or recognized translation, or
- add the term to the glossary for a more detailed description and quote the words in the original text.

Line breaks will cause spaces to be rendered in the text, which is unreasonable for Chinese content, as this section also points out in the above guideline.

Reply:
fixed, thx!

Reply:
It seems you added the `<td>...</td>` at the new codes at Line 27, but I surmise that it should replace Line 16-Line 19 instead.